stockpile/roles/postgresql/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+  # Collect PostgreSQL related configuration
+  - name: Detect the location for postgres configuration
+    shell:
+       find / -type f -name "postgresql.conf" | head -1
+    register: postgresql_config_file
```

Comment:
I'd suggest an uniform naming convention, the rest variables use pgsql, perhaps use the same here and rename it to pgsql_config_file? It'll make it easier while scraping and putting data back together